### ISSUE TITLE
Latest Version Download Changed

### ISSUE THREAD

**Issue by mlrobinson** [zdia/gorilla#147]
I was trying to install the latest password-gorilla via homebrew cask, and received an error. In debugging my error, it seems that the Hash of the file has changed from what homebrew is expecting. I checked here, and I don't see anything saying that the latest version (1.5.3.7.3) should have been updated since 2014. Just wondering if this file might have been updated and wasn't marked on the wiki. The file is verified vs the GPG signature, just making sure this is correct.

http://gorilla.dp100.com/downloads/gorilla.mac.15373.zip

**Comment by rich123** [comment]
Yes, the file was updated Jan 1, 2016.  The issue comment about the update is here: https://github.com/zdia/gorilla/issues/130#issuecomment-168367571

The Unix permissions that were encoded inside the zip file that is the Mac install bundle were not set properly, and the bundle, when installing, was throwing errors (you can read back in that thread for more details).

Since the only change was to file permissions, I did not bump the version number.  Of course I also was not expecting that homebrew had captured a hash of the file either.

FWIW, the sha256 hash of the zip on the download area is 51c443fb58a3628c2a45bd3160096abb9b017f33e6a08628636168f996ad0414 but as you have already verified the GPG signature, you should get this same sha256 hash value from the zip file.

**Comment by mlrobinson** [comment]
Thanks for the update. That is great to know. I can submit the change to fix up homebrew now, just wanted to verify it was correct before I went trying to do that.

**Comment by acidjunk** [comment]
Download links are not working?

**Comment by rich123** [comment]
Which links specifically?

**Comment by rich123** [comment]
The dp100.com domain server is back up now.
